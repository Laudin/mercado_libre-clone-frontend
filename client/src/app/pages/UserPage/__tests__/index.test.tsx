import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { UserPage } from '../index';

const shallowRenderer = createRenderer();

describe('<PublishPage />', () => {
   it('should match snapshot', () => {
      shallowRenderer.render(<UserPage />);
      const renderedOutput = shallowRenderer.getRenderOutput();
      expect(renderedOutput).toMatchSnapshot();
   });


});