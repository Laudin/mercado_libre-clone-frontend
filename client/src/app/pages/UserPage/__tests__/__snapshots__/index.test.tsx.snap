// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PublishPage /> should match snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
    prioritizeSeoTags={false}
  >
    <title>
      Perfil
    </title>
    <meta
      content=""
      name=""
    />
  </Helmet>
  <UserPage__Wrapper>
    <UserPage__PerfilWrapper>
      <UserPage__Perfil>
        <UserPage__ImageWrapper>
          <UserPage__Image
            src="http://localhost:3001/static/user_blue.png"
          />
        </UserPage__ImageWrapper>
        <UserPage__Info>
          <h2 />
          <p />
        </UserPage__Info>
      </UserPage__Perfil>
      <UserPage__ElemWrapper>
        <UserPage__Elem>
          Mis datos
        </UserPage__Elem>
      </UserPage__ElemWrapper>
      <UserPage__ElemWrapper>
        <UserPage__Elem>
          Seguridad
        </UserPage__Elem>
      </UserPage__ElemWrapper>
    </UserPage__PerfilWrapper>
  </UserPage__Wrapper>
</React.Fragment>
`;
